opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/decorator/ServletFilterHeaderSpanDecorator.java

```diff
@@ -0,0 +1,95 @@
+package io.opentracing.contrib.web.servlet.filter.decorator;
+
+import io.opentracing.Span;
+import io.opentracing.contrib.web.servlet.filter.ServletFilterSpanDecorator;
+import io.opentracing.tag.StringTag;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ServletFilterHeaderSpanDecorator implements ServletFilterSpanDecorator {
```

Comment:
This could be in a separate PR

Reply:
If we decide to split MDC and Headers, yes

Reply:
Done